### PR TITLE
Awesome books: with ES6

### PR DESCRIPTION
**Pull request by David-Kasilia**
### **To Highlight**

- Created a new directory and initialize it with git.
- Created the necessary files and directories required
- Created a branch to work on the es6 syntax and refactor code.
- Divided the code into necessary modules.
- Worked on the date and time using luxon
- Fixed the linters errors
- Updated the README file

### CODE REVIEW

File: modules/storebooks.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    books.forEach((book, index) => {
+      if (book.title === title) {
+        books.splice(index, 1);
+      }
+    });
+
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+}
```
Comment (DammyShittu) [inline modules/storebooks.js]: - [x] Your app does not really clear the local storage on removing the books. When I remove books and reload all the books still come back on the page. Kindly fix this.

  books are added | books are removed | Page is refreshed
  :---:|:---:|:---:
  awe-1 | awe-2 | awe-1

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @David-Kasilia,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Arrow functions

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/storebooks.js (hunk 1 above)
Comment (David-Kasilia) [reply modules/storebooks.js]: Thank You for your Review I have Fixed the issue.

**Review by codecaiine — APPROVED** [review]
Hi @David-Kasilia , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Descriptive Pull Request :heavy_check_mark: 
- Good use of gitflow :heavy_check_mark: 
- Clean code :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**